docs/v0.7/en/user-guide/operations/configuration.md

```diff
@@ -112,13 +112,16 @@ greptime standalone start --help
 ### Examples
 
 Configurations can be used in one or multiple components according to their features.
+This document only contains examples for frequently used configurations. Please refer to the auto-generated [document](https://github.com/GreptimeTeam/greptimedb/blob/main/config/config.md) for all available configurations.
```

Comment:
It's better to refer to a specific tag link. We can use `{version}` to get the current document version? @nicecui

Reply:
Not yet, I will discuss this requirement with @Aganivi as soon as possible.

Reply:
This feature should be implemented ASAP. I remember it was discussed many times.

Reply:
Got it, ASAP